client/src/Helper.ts

```diff
@@ -124,12 +126,15 @@ export class Helper {
             reject(err.message);
           } else {
             if (!Array.isArray(javaHomes) || javaHomes.length === 0) {
-              Helper.log("Could not find Java home");
-              reject("no Java home found");
+              const msg = "Could not find a 64-bit JDK with at least version 1.8. "
+                + "Please install one and/or manually specify it in the Gobra settings.";
+              Helper.log(msg);
+              vscode.window.showErrorMessage(msg);
+              reject(msg);
```

Comment:
This is most likely the wrong code section, but, if the IDE is using JAVA_HOME or something similar to get the location of the Java runtime, then could it output the path that the IDE used in case the JAVA version is bad. We sometimes have the problem that we do not know which Java version the IDE used for people who have multiple java versions installed.

Reply:
There's a new IDE setting `gobraDependencies.java.javaBinary` that allows you to hard-code the path to the java binary. However, I agree that we have had issues with choosing the wrong Java binary in the past. How do you envision such a feature to show the chosen java path? Should I add a command "Show Java Path" that does nothing else then showing an info popup with the path and maybe some additional information how this one was chosen (e.g. automatically detected vs. manually set in settings)?

Reply:
yeah, exactly. I mean it is not urgent in any way, but I think it will be useful when we have other people instal Gobra.

Reply:
I've created issue #17 for this